src/javascript/EditPanel/header/UpperSection.jsx

```diff
@@ -70,8 +70,8 @@ export const HeaderUpperSection = ({title, actionContext}) => {
                                 render={ButtonRenderer}
                             />
 
-                    <Typography isNowrap className={styles.headerTypography} variant="heading">
-                        {title}
+                    <Typography isNowrap className={styles.headerTypography} variant="title">
+                        {truncate(title, 60)}
```

Comment:
why limiting text size? the **isNowrap** must do it

Reply:
limited as asked in the ticket, confirmation asked to Romain